src/pages/verify/e-ids/danish-mitid.mdx

```diff
@@ -39,6 +39,25 @@ You create personal MitID test users at [https://pp.mitid.dk/test-tool/frontend]
 
 ![MitID test tool](./images/mitid-testtool.png)
 
+## Available data / scopes
```

Comment:
I think the headline should be something like `Collecting user addresses` instead. (For NemID as well)

Reply:
Updated